tools/fota_delta/mkfotadeltatool.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+###########################################################################
+#
+# Copyright 2019 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+
+echo "Making FOTA Delta tool"
+
+# Environmental Stuff
+
+FOTA_DELTA_TOOL_PATH=`test -d ${0%/*} && cd ${0%/*}; pwd`
+#When location of this script is changed, only OS_PATH should be changed !!!
+OS_PATH=$FOTA_DELTA_TOOL_PATH/../../os
+SRCDIR=$FOTA_DELTA_TOOL_PATH/src
+SRC_INC_DIR=$OS_PATH/include
+DEST_INC_DIR=$FOTA_DELTA_TOOL_PATH/include
+
+#load .config file
+source $OS_PATH/.config
+
+APPNAME=mkfotadelta
```

Comment:
Could you let me know the git status after building? Doesn't it show that it is a change?
As I remember nxfuse does. Could you check it and talk the person in charge of nxfuse as well?